src/wx/strutils.h

```diff
@@ -7,15 +7,23 @@
 #include <wx/string.h>
 #include <wx/arrstr.h>
 
+namespace strutils
+{
```

Comment:
I'm used to putting the opening curly brace on the same line, it saves on vertical space. I don't feel very strongly on the matter, though.